CHANGELOG.md

```diff
@@ -5,22 +5,32 @@
 
 ### Enhancements
 * Exporting a `RealmEventName` type. ([#6300](https://github.com/realm/realm-js/pull/6300))
+* Automatic client reset recovery now preserves the original division of changesets, rather than combining all unsynchronized changes into a single changeset ([realm/realm-core#7161](https://github.com/realm/realm-core/pull/7161)).
```

Comment:
```suggestion
* Automatic client reset recovery now preserves the original division of changesets, rather than combining all unsynchronized changes into a single changeset. ([realm/realm-core#7161](https://github.com/realm/realm-core/pull/7161))
```